addons/web_editor/static/src/scss/secondary_variables.scss

```diff
@@ -48,18 +48,7 @@ $o-theme-color-palette: map-merge($o-base-theme-color-palette, o-map-omit($o-the
 // white-50 but does not allow overridding that with variables).
 $o-gray-color-palette: o-safe-nth($o-gray-color-palettes, $o-gray-color-palette-number, ()) !default;
 $o-gray-color-palette: map-merge($o-transparent-grays, o-map-omit($o-gray-color-palette));
-$gray-hue: 210 !default;
-$gray-extra-saturation: 0 !default;
-@function adjust-gray-color($gray) {
-    @return adjust-color(change-color($gray, $hue: $gray-hue * 1deg), $saturation: $gray-extra-saturation * 1%);
-}
-$o-adjusted-gray-color-palette: ();
-@each $key, $color in $o-base-gray-color-palette {
-    $o-adjusted-gray-color-palette: map-merge($o-adjusted-gray-color-palette,
-            (base-#{$key}: $color, $key: adjust-gray-color($color))
-    );
-}
-$o-gray-color-palette: map-merge($o-adjusted-gray-color-palette, o-map-omit($o-gray-color-palette));
```

Comment:
OK your first commit creates this, your second removes it... it makes review complicated and history useless. Try to refactore your history so that it is clear.